packages/datatrak-web/src/components/Toast/ToastProvider.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import { SnackbarProvider } from 'notistack';
+import { Toast } from './Toast';
+
+export const ToastProvider = ({ children }: { children: React.ReactNode }) => {
```

Comment:
hmmm I'm not 100% sure but I wouldn't think to look in components for the provider. Maybe this should just go in the AppProviders file? Or in a new Providers directory if we want to split out this provider and the react query provider?